### PR TITLE
Feature/kitosudv 2300 delta

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/Controllers/API/V2/External/ItInterfaces/ItInterfaceV2Controller.cs [hunk 1]
```diff
@@ -85,22 +85,26 @@ public IHttpActionResult PostItInterfaceAsRightsHolder([FromBody] RightsHolderCr
         [SwaggerResponse(HttpStatusCode.Forbidden)]
         public IHttpActionResult GetItInterfacesAsRightsHolder(
             [NonEmptyGuid] Guid? rightsHolderUuid = null,
-            bool includeDeactivated = false,
+            bool? includeDeactivated = null,
```
Comment (JacobMalling) [inline Presentation.Web/Controllers/API/V2/External/ItInterfaces/ItInterfaceV2Controller.cs:88]: Fint. Så minder den mere om de andre optional parametre 👍

**Review by JacobMalling — APPROVED** [review]
Det ser jo meget godt ud

File: Presentation.Web/Controllers/API/V1/OData/AttachedOptions/AttachedOptionsController.cs [hunk 2]
```diff
@@ -33,5 +35,26 @@ private IHasAttachedOptions GetOwner(EntityType optionObjectType, int optionObje
...
+        }
+
+        protected override void RaiseUpdatedDomainEvent(AttachedOption entity)
+        {
+            RaiseRootUpdated(entity);
+        }
+
+        private void RaiseRootUpdated(AttachedOption entity)
+        {
+            var itSystemUsage = _usageRepository.GetSystemUsage(entity.ObjectId);
```
Comment (mrjsawdk) [inline Presentation.Web/Controllers/API/V1/OData/AttachedOptions/AttachedOptionsController.cs:56]: mangler check på null

File: Tests.Integration.Presentation.Web/Tools/ItSystemHelper.cs [hunk 3]
```diff
@@ -132,6 +132,17 @@ public static async Task<HttpResponseMessage> DeleteItSystemAsync(int systemId,
             return await HttpApi.PatchWithCookieAsync(url, cookie, body);
         }
 
+        public static async Task<ItSystemDTO> SetNameRequestAsync(
```
Comment (mrjsawdk) [inline Tests.Integration.Presentation.Web/Tools/ItSystemHelper.cs:135]: SetNameAsync

**Pull request merged**